src/lib/pages/frontsite/PatronProfile/PatronPastDocumentRequests/PatronPastDocumentRequests.js

```diff
@@ -59,11 +59,24 @@ class PatronPastDocumentRequests extends Component {
     if (row.metadata.state !== 'ACCEPTED') {
       return _startCase(row.metadata.state.toLowerCase());
     }
-    return (
-      <Link to={FrontSiteRoutes.documentDetailsFor(row.metadata.document_pid)}>
-        <LiteratureTitle title={row.metadata.document.title} />
-      </Link>
-    );
+    return <LiteratureTitle title={row.metadata.document.title} />;
+  };
+
+  searchLinkFormatter = ({ row }) => {
+    return row.metadata.state === 'ACCEPTED' ? (
+      <Popup
+        content="Search for this literature in the catalogue. Note that it may no longer be available."
+        trigger={
+          <Link
+            to={FrontSiteRoutes.documentsListWithQuery(
+              row.metadata.document.title
+            )}
+          >
+            <Icon name="search" />
+          </Link>
+        }
+      />
```

Comment:
Minor: This popup is almost duplicated above as well, maybe it's worth to create a small cmp ro reuse it instead of duplicate it